### PR TITLE
[typescript/en] Fix formatting, close #990

### LINKED ISSUE #990: comment incorrect at var p: Person

### PR DESCRIPTION
**Pull request by NPrescott**
Cleaned up mixed tabs/spaces
Wrapped lines at 80 characters
Fixed incorrect comment regarding property name

I took some liberty in cleaning things up while I was fixing #990 (fixed [here](https://github.com/NPrescott/learnxinyminutes-docs/commit/2ae3d41c2749645fec4748c764ccd5c9405424e5#diff-2a430f897f1d70dad08d5561446c10caR66)). I modified the spacing after `//` to be more in line with [Microsoft's examples](https://github.com/Microsoft/TypeScriptSamples).

### CODE REVIEW

File: typescript.html.markdown [hunk 1]
```diff
@@ -14,100 +14,111 @@ This article will focus only on TypeScript extra syntax, as oposed to [JavaScrip
...
+  // And of course functions
   move(): void;
 }
 
-//..Object that implements the "Person" interface
-var p : Person = { name: "Bobby", move : () => {} }; //Can be treated as a Person since it has the name and age properties
-//..Objects that have the optional property:
+// Object that implements the "Person" interface
+// Can be treated as a Person since it has the name and move properties
+var p : Person = { name: "Bobby", move : () => {} };
```
Comment (geoffliu) [inline typescript.html.markdown]: Is it customary to have a space between the variable and the colon? The rest of this page seems to not have spaces.

File: typescript.html.markdown [hunk 2]
```diff
@@ -14,100 +14,111 @@ This article will focus only on TypeScript extra syntax, as oposed to [JavaScrip
...
+// Objects that have the optional property:
 var validPerson : Person = { name: "Bobby", age: 42, move: () => {} };
```
Comment (geoffliu) [inline typescript.html.markdown]: Same here

File: typescript.html.markdown [hunk 3]
```diff
@@ -14,100 +14,111 @@ This article will focus only on TypeScript extra syntax, as oposed to [JavaScrip
...
-var invalidPerson : Person = { name: "Bobby", age: true }; //Is not a person because age is not a number
+// Is not a person because age is not a number
+var invalidPerson : Person = { name: "Bobby", age: true };
```
Comment (geoffliu) [inline typescript.html.markdown]: And here

File: typescript.html.markdown (hunk 1 above)
Comment (geoffliu) [inline typescript.html.markdown]: Looks like Microsoft's examples never have space before colon. Let's just do a `s/ :/:/`

**Comment by geoffliu** [comment]
Thanks for going over this page and picking out the problems! Just a few left over nits to pick, and we'll merge it.

**Comment by NPrescott** [comment]
Suggested changes made, ready whenever.

**Comment by geoffliu** [comment]
Wonderful, thanks!

**Pull request merged**